packit_service/worker/allowlist.py

```diff
@@ -67,22 +73,90 @@ def _strip_protocol_and_add_git(url: Optional[str]) -> Optional[str]:
             return None
         return url.split("://")[1] + ".git"
 
-    @staticmethod
-    def add_namespace(namespace: str) -> bool:
+    def init_kerberos_ticket(self):
+        """
+        Try to init kerberos ticket.
+
+        Returns:
+            Whether the initialisation was successful.
+        """
+        try:
+            logger.debug("Initialising Kerberos ticket so that we can use fasjson API.")
+            PackitAPI(
+                config=self.service_config, package_config=None
+            ).init_kerberos_ticket()
+        except PackitCommandFailedError as ex:
+            msg = f"Kerberos authentication error: {ex.stderr_output}"
+            logger.error(msg)
+            return False
+
+        return True
+
+    def is_github_username_from_fas_account_matching(self, fas_account, sender_login):
+        """
+        Compares the Github username from the FAS account
+        to the username of the one who triggered the installation.
+
+        Args:
+            fas_account: FAS account for which we will get the account info.
+            sender_login: Login of the user that will be checked for be match
+                            against info from FAS.
+
+        Returns:
+            Whether there was a match found.
+        """
+        if not self.init_kerberos_ticket():
+            return False
```

Comment:
I would at least mention in docstring that failure means `False` as well. (For me, it probably does not fit the method name `Is GitHub username from FAS account matching?`.)


I understand why it is like this... Maybe, we can check the errors outside of the method? (Just thinking out loud...;)